src/platform/apollolake/include/platform/lib/memory.h

```diff
@@ -292,7 +292,7 @@
 
```

Comment:
@lyakh what is the problem are we trying to solve. One very interesting things that I noticed on IMX and Cavs platforms is PLATFORM_DCACHE_ALIGN.

Runtime memory is always aligned and PLATFORM_DCACHE_ALIGN can be as much as 128 or 64 this means blocks with sizes less than 128 or 64 are rarely used on cAVS and IMX platforms.

This was actually a problem for iMX where we quickly run out of memory. On iMX we solved the problem by reducing PLATFORM_DCACHE_ALIGN to sizeof(void*). Maybe this is helpful for your problem.

But the main question here is WHY do we need this change now and how did you get to create such patch.

Reply:
@dbaluta this patch isn't for arbitrary allocations, it's for DMA buffers. I added debug prints and I see 0x180 and 0x300 byte large DMA buffers being allocated. So, every time a 0x180 byte buffer is allocated, two 0x100 byte large chunks are occupied, which means 0x80 bytes are wasted. That's exactly the reason (I assume) why a non-intuitive size of 0x180 bytes was used before #2425

Reply:
@lyakh OK, I see. I'm OK with the patch. Out of curiosity is there a bug associated with this?